Content.Shared/Weather/WeatherComponent.cs

```diff
@@ -29,19 +28,19 @@ public sealed partial class WeatherData
     /// <summary>
     /// When the weather started if relevant.
     /// </summary>
-    [ViewVariables, DataField("startTime", customTypeSerializer: typeof(TimeOffsetSerializer))]
+    [DataField(customTypeSerializer: typeof(TimeOffsetSerializer))] //TODO: Remove Custom serializer
```

Comment:
Should be fine to remove these now?

Reply:
![image](https://github.com/space-wizards/space-station-14/assets/96445749/8179df94-3614-467a-8b00-70a958274afd)
I'm just not sure about this, so I was waiting for advice from sloth, but I often don't fully understand what he means

Reply:
Oh, I getcha. One sec.

Reply:
its not fine i think